build.gradle

```diff
@@ -179,6 +180,27 @@ project(":samza-core_$scalaVersion") {
   }
 }
 
+
+project(':samza-azure') {
+  apply plugin: 'java'
+  apply plugin: 'checkstyle'
+
+  dependencies {
+    compile "com.microsoft.azure:azure-storage:5.3.1"
+    compile "com.fasterxml.jackson.core:jackson-core:2.8.8"
+    compile project(':samza-api')
+    compile project(":samza-core_$scalaVersion")
```

Comment:
Samza core has the JobCoordinator, LeaderElector and a few other interfaces that I'm implementing for Azure.